src/zarate.jl

```diff
@@ -39,6 +39,8 @@ more crossings than optimal : edges should instead be routed through these diffe
 `paths` contains for each edge, a Tuple of vectors, representing that route through the
 different nodes as x and y coordinates.
 
+opt_node_order: optional dictionary stating optional arguments for the nodes ordering in each layer
```

Comment:
The should probably be called `node_order`.
and should make it clear how to specify things, and what happens if an element is not specied.

Reply:
I updated the file using force_order as optional argument